src/modules/ejector/data_encode.py

```diff
@@ -0,0 +1,32 @@
+from eth_typing import HexStr
+
+from src.utils.types import hex_str_to_bytes
+from src.web3py.extentions.lido_validators import NodeOperatorGlobalIndex, LidoValidator
+
+
+DATA_FORMAT_LIST = 1
+
+MODULE_ID_LENGTH = 3
+NODE_OPERATOR_ID_LENGTH = 5
+VALIDATOR_INDEX_LENGTH = 8
+VALIDATOR_PUB_KEY_LENGTH = 48
+
+
+def encode_data(validators: list[tuple[NodeOperatorGlobalIndex, LidoValidator]]):
+    #     /// MSB <------------------------------------------------------- LSB
+    #     /// |  3 bytes   |  5 bytes   |     8 bytes      |    48 bytes     |
+    #     /// |  moduleId  |  nodeOpId  |  validatorIndex  | validatorPubkey |
+
+    result = b''
+
+    for (module_id, op_id), validator in validators:
+        result += module_id.to_bytes(MODULE_ID_LENGTH)
+        result += op_id.to_bytes(NODE_OPERATOR_ID_LENGTH)
+        result += int(validator.index).to_bytes(VALIDATOR_INDEX_LENGTH)
+
+        if len(validator.validator.pubkey) != VALIDATOR_PUB_KEY_LENGTH:
```

Comment:
according to `pubkey` type here it's a string, so its length will be = 98 
2 ('0x') + 48 bytes * 2 = 98